src/components/NavBar/NavBar.js

```diff
@@ -75,11 +75,18 @@ const NavBar = () => {
           </Link>
         </Menu.Item>
         {authState.isAuthenticated && (
-          <Menu.Item key="5">
-            <div className="menu-link" onClick={logout}>
-              Log out
-            </div>
-          </Menu.Item>
+          <>
```

Comment:
Hmmm so I think all the changes up until this point make sense, but isn't this moving these buttons / actions to the navbar in the header? If so, I think that's what Jason was saying they don't want, ie they don't want it to be prominent.

Reply:
it's conditional on this authState.isAuthenticated, so it only shows up if the user is logged in. A non admin shouldn't be able to see the buttons.

Reply:
Great, got it